### ISSUE TITLE
Sandbox for Lists

### ISSUE THREAD

**Issue by Tasilee** [AtlasOfLivingAustralia/specieslist-webapp#101]
I raised this in my May 2018 report and adding here so it is 'on the books'.

A 'smart' Sandbox for importing species lists with traits. Need 'AI' support to query and display of relevant terms from vocabularies and ontologies to align data with best current practice. Key aspects of this (database) are items name, definition, link up, link down, date agreed, date added, reference link etc. This strategy builds efficiently on what we currently have or know about, can tap the best practice trait terms and just as importantly, identify gaps. 

Traitbank?

In short, column headers on List matrices are looked up in a database and best matches returned with definitions, source for acceptance/rejection. New terms from headers added but need validation by community of experts.
